performance-test/README.md

```
@@ -1,178 +1,103 @@
-			= 600
-
-* Dropdown option in Constant Throughput Timer
-	* Calculate Throughput based on as = All active threads in current thread group
-		* If we are performing load test with 10TPS as hits / sec in one thread group. Then we need to provide value hits / minute as in Constant Throughput Timer
-	 			Value = 10 X 60
-					  = 600
-		  
-	* Calculate Throughput based on as = this thread
-		* If we are performing scalability testing we need to calculate throughput for 10 TPS as 
-          Value = (10 * 60 )/(Number of users)
+This module describes how to conduct load test of the eSignet OIDC/DPoP authentication flow (OTP-based and Biometric-based) using the provided JMeter script (`eSignetGo_script_v0.3.jmx`).
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)